src/ArduinoIoTCloudTCP.cpp

```diff
@@ -763,14 +763,16 @@ void ArduinoIoTCloudTCP::sendPropertiesToCloud()
 }
 
 #if OTA_ENABLED
-void ArduinoIoTCloudTCP::sendOTAPropertiesToCloud()
+void ArduinoIoTCloudTCP::sendOTAPropertiesToCloud(bool include_ota_req)
 {
   PropertyContainer ota_property_container;
   unsigned int last_ota_property_index = 0;
 
-  std::list<String> const ota_property_list {"OTA_CAP", "OTA_ERROR", "OTA_SHA256"};
-  std::for_each(ota_property_list.cbegin(),
-                ota_property_list.cend(),
+  std::list<String> ota_property_list {"OTA_CAP", "OTA_ERROR", "OTA_SHA256"};
+  if (include_ota_req)
```

Comment:
I'd suggest to either enclose the expression below within braces, i.e.
```C++
if (include_ota_req) {
  ota_property_list.push_back("OTA_REQ");
}
```
or to leave one free line below the statement before and the statement after. Otherwise one get's all expressions clustered up and its not easy to see what belongs to the if statement and what not.

Reply:
fully agree, this was part of the initial changes i've done to support OTA with this new flow, then i've changed this function in this commit https://github.com/arduino-libraries/ArduinoIoTCloud/pull/297/commits/e6b252c280d27929e9d0eeb1995d4cf29dd624e1 and again here https://github.com/arduino-libraries/ArduinoIoTCloud/pull/297/commits/98ed735b7136c7acbc22cd633424848be62da1d4